packages/perseus/src/widgets/table/validate-table.ts

```diff
@@ -0,0 +1,24 @@
+import {filterNonEmpty} from "./utils";
+
+import type {PerseusScore} from "../../types";
+import type {PerseusTableUserInput} from "../../validation.types";
+
+export function validateTable(state: PerseusTableUserInput): PerseusScore {
```

Comment:
When we refined types, we started referring to `state` as `userInput`. Would you be up for changing it here to follow that naming convention? 

I think you might have said at one point you preferred calling it state, though I could be misremembering, so if that is the better option, we might want to go back and change them all to be state to be consistent. `scoreTable` is still showing `state` because it got skipped during the refinement step (whoops)

Reply:
If the type is `XyzUserInput` I think calling the variable or parameter `userInput` makes alot of sense. :)   I'll update this PR.

Reply:
Thank you! Also noticed that this isn't using a default export like the scoring functions do. Was that on purpose?

Reply:
Oops, I'll fix that too! Nice catch.